charts/kof-storage/templates/victoria/service-monitor.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.victoriametrics.enabled }}
```

Comment:
Probably not needed as this will be a duplication with kof-collectors if this be moved there.